go/vt/vtgate/vtgate.go

```diff
@@ -179,6 +179,8 @@ var (
 	// catch the initial load stats.
 	vschemaCounters = stats.NewCountersWithSingleLabel("VtgateVSchemaCounts", "Vtgate vschema counts", "changes")
 
+	vschemaVindexUnknownParams = stats.NewGauge("VSchemaVindexUnknownParams", "Number of params unrecognized by VSchema Vindexes")
```

Comment:
nit: `stats.NewGauge("VindexUnknownParameters", "Number of parameters unrecognized by Vindexes")`